main.tf

```diff
@@ -137,7 +137,7 @@ data "aws_iam_policy_document" "datadog_integration_policy" {
 }
 
 module "datadog_integration_role" {
-  source        = "github.com/schubergphilis/terraform-aws-mcaf-role?ref=v0.3.2"
+  source        = "github.com/schubergphilis/terraform-aws-mcaf-role?ref=v0.4.0"
```

Comment:
While you're busy anyway could you change this to?

https://registry.terraform.io/modules/schubergphilis/mcaf-role/aws/latest?tab=outputs

  source  = "schubergphilis/mcaf-role/aws"
  version = "~> 0.4.0"